src/main/java/edu/cornell/kfs/module/purap/document/validation/impl/CuCreditMemoDocumentPreRules.java

```diff
@@ -97,37 +97,37 @@ protected boolean checkWireTransferTabState(final VendorCreditMemoDocument cmDoc
         return tabStatesOK;
     }
 
-    protected boolean hasWireTransferValues(final CreditMemoWireTransfer cmWireTransfer) {
+    protected boolean hasWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {
         boolean hasValues = false;
 
         // Checks each explicit field in the tab for user entered values
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmAutomatedClearingHouseProfileNumber());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmBankName());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmBankRoutingNumber());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmBankCityName());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmBankStateCode());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmBankCountryCode());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmPayeeAccountNumber());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmAttentionLineText());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmCurrencyTypeName());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmAdditionalWireText());
-        hasValues |= StringUtils.isNotBlank(cmWireTransfer.getCmPayeeAccountName());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getAutomatedClearingHouseProfileNumber());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getBankName());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getBankRoutingNumber());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getBankCityName());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getBankStateCode());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getBankCountryCode());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getPayeeAccountNumber());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getAttentionLineText());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getCurrencyTypeName());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getAdditionalWireText());
+        hasValues |= StringUtils.isNotBlank(wireTransfer.getPayeeAccountName());
 
         return hasValues;
     }
     
-    protected void clearWireTransferValues(final CreditMemoWireTransfer cmWireTransfer) {
-    	cmWireTransfer.setCmAutomatedClearingHouseProfileNumber(null);
-    	cmWireTransfer.setCmBankName(null);
-    	cmWireTransfer.setCmBankRoutingNumber(null);
-    	cmWireTransfer.setCmBankCityName(null);
-    	cmWireTransfer.setCmBankStateCode(null);
-    	cmWireTransfer.setCmBankCountryCode(null);
-    	cmWireTransfer.setCmPayeeAccountNumber(null);
-    	cmWireTransfer.setCmAttentionLineText(null);
-    	cmWireTransfer.setCmCurrencyTypeName(null);
-    	cmWireTransfer.setCmAdditionalWireText(null);
-    	cmWireTransfer.setCmPayeeAccountName(null);
+    protected void clearWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {

```

Comment:
Another method that could be replaced with `PaymentSourcePreRulesService` usage.

Reply:
Another method for which the `PaymentSourcePreRuleServiceImpl` variant might be getting used instead.